tests/scene/test_camera_2d.h

```diff
@@ -0,0 +1,318 @@
+	}
+
+	SUBCASE("Drawing") {
+		test_camera->set_margin_drawing_enabled(true);
+		CHECK(test_camera->is_margin_drawing_enabled());
+		test_camera->set_margin_drawing_enabled(false);
+		CHECK_FALSE(test_camera->is_margin_drawing_enabled());
+		test_camera->set_limit_drawing_enabled(true);
+		CHECK(test_camera->is_limit_drawing_enabled());
+		test_camera->set_limit_drawing_enabled(false);
+		CHECK_FALSE(test_camera->is_limit_drawing_enabled());
+		test_camera->set_screen_drawing_enabled(true);
+		CHECK(test_camera->is_screen_drawing_enabled());
+		test_camera->set_screen_drawing_enabled(false);
+		CHECK_FALSE(test_camera->is_screen_drawing_enabled());
+	}
+
+	SUBCASE("Enabled") {
+		test_camera->set_enabled(true);
+		CHECK(test_camera->is_enabled());
+		test_camera->set_enabled(false);
+		CHECK_FALSE(test_camera->is_enabled());
+	}
+
+	SUBCASE("Rotation") {
+		constexpr float rotation_smoothing_speed = 20.0f;
+		test_camera->set_ignore_rotation(true);
+		CHECK(test_camera->is_ignoring_rotation());
+		test_camera->set_ignore_rotation(false);
+		CHECK_FALSE(test_camera->is_ignoring_rotation());
+		test_camera->set_rotation_smoothing_enabled(true);
+		CHECK(test_camera->is_rotation_smoothing_enabled());
+		test_camera->set_rotation_smoothing_speed(rotation_smoothing_speed);
+		CHECK(test_camera->get_rotation_smoothing_speed() == rotation_smoothing_speed);
+	}
+
+	SUBCASE("Zoom") {
+		const Vector2 zoom = Vector2(4, 4);
+		test_camera->set_zoom(zoom);
+		CHECK(test_camera->get_zoom() == zoom);
+	}
+
+	SUBCASE("Offset") {
+		const Vector2 offset = Vector2(100, 100);
+		test_camera->set_offset(offset);
+		CHECK(test_camera->get_offset() == offset);
+	}
+
+	SUBCASE("Limit") {
+		constexpr int limit_left = 100;
+		constexpr int limit_top = 100;
+		constexpr int limit_right = 100;
+		constexpr int limit_bottom = 100;
+		test_camera->set_limit_smoothing_enabled(true);
+		CHECK(test_camera->is_limit_smoothing_enabled());
+		test_camera->set_limit_smoothing_enabled(false);
+		CHECK_FALSE(test_camera->is_limit_smoothing_enabled());
+		test_camera->set_limit(SIDE_LEFT, limit_left);
+		CHECK(test_camera->get_limit(SIDE_LEFT) == limit_left);
+		test_camera->set_limit(SIDE_TOP, limit_top);
+		CHECK(test_camera->get_limit(SIDE_TOP) == limit_top);
+		test_camera->set_limit(SIDE_RIGHT, limit_right);
+		CHECK(test_camera->get_limit(SIDE_RIGHT) == limit_right);
+		test_camera->set_limit(SIDE_BOTTOM, limit_bottom);
+		CHECK(test_camera->get_limit(SIDE_BOTTOM) == limit_bottom);
+	}
+
+	SUBCASE("Position") {
+		constexpr float smoothing_speed = 20.0f;
+		test_camera->set_position_smoothing_enabled(true);
+		CHECK(test_camera->is_position_smoothing_enabled());
+		test_camera->set_position_smoothing_speed(smoothing_speed);
+		CHECK(test_camera->get_position_smoothing_speed() == smoothing_speed);
+	}
+
+	memdelete(test_camera);
+}
+
+TEST_CASE("[SceneTree][Camera2D] Camera positioning") {
+	SubViewport *test_viewport = memnew(SubViewport);
```

Comment:
I noticed Camera3D test calls this variable `mock_viewport` instead of `test_viewport`. I suggest using the same name for consistency (it's called "mock" because this viewport is not the subject of the unit test).

Reply:
Done!